README.md

```diff
@@ -23,44 +23,49 @@ new [Vue Composition API](https://v3.vuejs.org/guide/composition-api-introductio
 ```typescript
 import { createApp } from 'vue';
 import CasbinPlugin from 'vue-authz';
-import { Authorizer } from 'casbin.js';
-
-const permission = {
-    "read": ['data1', 'data2'],
-    "write": ['data1']
-}
-
-// Run casbin.js in manual mode, which requires you to set the permission manually.
-const authorizer = new casbinjs.Authorizer("manual");
-
-authorizer.setPermission(permission);
+import { newEnforcer, MemoryAdapter } from 'casbin.js';
+
+const a = new MemoryAdapter("p, alice, data1, read\n" +
+    "p, alice, data2, read\n" +
+    "p, alice, data2, write\n" +
+    "p, bob, data2, write")
+
+enforcer = await newEnforcer("[request_definition]\n" +
```

Comment:
how to use the adapter?

Reply:
The usage of the `MemoryAdapter` is quite confusing, which shows an unused variable in the context. If `MemoryAdapter` initializes some global contexts, please add some notifications or comments in your docs.